### DISCUSSION TITLE
Is this possible to enjoy refresh/reload for translations?

### DISCUSSION THREAD

**Discussion by MoOx** [i18next/next-i18next#1811] [Q&A] [answered]
I have my text in json files (I followed the doc) but when I want to add new entries, I need to kill & restart next dev server.
Is there a way to have files refreshed without this restart? I would have expected at least a refresh when there is a full page reload.

Am I doing something wrong? Any idea?

**Comment by isaachinman** [comment, marked as answer]
https://github.com/isaachinman/next-i18next/issues/1264#issuecomment-863846792
